### DISCUSSION TITLE
Gowitness should raise DOM as HTTP_RESPONSE

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#713] [Ideas]

**Comment by TheTechromancer** [comment]
Upstream issue opened here: https://github.com/sensepost/gowitness/issues/190. We are waiting to implement this until we can get content of the network logs as well.

Gowitness module TODO:
- [ ] Raise `HTTP_RESPONSE` for final rendered DOM
- [ ] Raise `HTTP_RESPONSE` for each network request
- [ ] Include perception hash in `WEBSCREENSHOT` event data

**Reply by leonjza** [reply]
v3 will have this soon™
